snippets/kickstart_networking_setup.erb

```diff
@@ -113,8 +116,14 @@ DEVICE="$real"
 HWADDR="<%= interface.mac -%>"
 <% end -%>
 ONBOOT=yes
+<% if interface.respond_to?(:primary) && interface.primary -%>
+DEFROUTE=yes
```

Comment:
I think that default route is usually determined by interface gateway which comes from DHCP. We can have more interfaces, each with different gateway. My understanding was that in such case DEFROUTE sets which gateway is used for default route (making it default gateway). I'm not 100% sure about all DEFROUTE impacts and there's not much documentation.